docs/mddocs/Quickstart/ollama_portablze_zip_quickstart.md

```diff
@@ -47,3 +47,38 @@ You could then use Ollama to run LLMs on Intel GPUs as follows:
 <div align="center">
   <img src="https://llm-assets.readthedocs.io/en/latest/_images/ollama_portable_run_ollama.png"  width=80%/>
 </div>
+
+## Runtime Configurations
```

Comment:
Instead of documenting "Runtime Configurations", we should probably provide solutions to specific problems:
1) Speed up model download using alternative sources
2) Increase context length in Ollama

Reply:
Thank you. Have updated :)